src/Autocomplete.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component, PropTypes } from 'react';
+import cx from 'classnames';
+import constants from './constants';
+import Icon from './Icon';
+import Row from './Row';
+import Col from './Col';
+
+class Autocomplete extends Component {
+  constructor (props) {
+    super(props);
+
+    this.state = {
+      value: '',
+      minLength: this.props.minLength
+    };
+
+    delete this.props.minLength;
```

Comment:
The props goes unused in the render function so unless I pass it as an argument to render the dropdown, ESLINT yells at me :(